src/controller/reviews.controller.js

```diff
@@ -0,0 +1,118 @@
+import Review from "../model/review.model.js";
+import { response } from "express";
+
+export const reviewsGet = async (req, res = response) => {
+  const { limite, desde } = req.query;
+  const query = {};
+
+  const [total, reviews] = await Promise.all([
+    Review.countDocuments(query),
+    Review.find(query).skip(Number(desde)).limit(Number(limite)),
+  ]);
+
+  res.status(200).json({
+    total,
+    reviews,
+  });
+};
+
+export const getReviewById = async (req, res) => {
+  const { id } = req.params;
+  const review = await Review.findById(id);
+  if (!review) {
+    return res.status(404).json({
+      msg: "Revisión no encontrada",
+    });
+  }
+
+  res.status(200).json({
+    review,
+  });
+};
+
+export const putReview = async (req, res = response) => {
+  const { id } = req.params;
+  const {
+    comment,
+    rating_cleanliness,
+    rating_staff,
+    rating_facilities,
+    tp_status,
+    is_custom,
+    user_id,
+    hotel_id,
+  } = req.body;
```

Comment:
`tp_status` NO DEBE ser pasado en el body, es necesario que elimines eso. ademas, tampoco tiene mucho sentido que puedas cambiar el hotel y usuario asociado. por lo que tampoco validaremos eso. ademas, `is_custom` no es una prop de nuestra entidad. es `is_customer`, porfavor arreglar eso

```suggestion
  const {
    comment,
    rating_cleanliness,
    rating_staff,
    rating_facilities,
    is_customer,
  } = req.body;
```